src/client/common/application/notebook.ts

```diff
@@ -39,11 +41,17 @@ export class VSCodeNotebook implements IVSCodeNotebook {
     }
     public get activeNotebookEditor(): NotebookEditor | undefined {
         if (!this.useNativeNb) {
+            console.error('Not using native');
             return;
         }
         try {
+            console.error(`window.visibleNotebookEditors.length = ${window.visibleNotebookEditors.length}`);
```

Comment:
To be removed.

Reply:
😆 I've got a PR with all the same logging in. Does this PR have a fix for the creation issue? I wasn't seeing that offhand. Checking since I was looking at that as well.

Reply:
Looks like more tests are failing because of this, i just found that out (my tests were failing because of this as well)
I thought my tests was failing due to some popup (but that wasn't the case), the notebook wasn't getting created.

Reply:
I had a PR here that does fix the verify cells test, but I think it's a hack. 
https://github.com/microsoft/vscode-jupyter/pull/6069/files
I explicitly check for one visible document being open, and then I force it active. But I'm not sure this is a good fix as I don't know yet what was making it inactive.

Reply:
At least in that case the notebook was getting created, but it's getting set off of active by something. Doesn't happen locally.

Reply:
Damn it, will need to debug this in VSCode and notify VSCode where the actual bug is